src/integrations/firebase.ts

```diff
@@ -58,9 +58,23 @@ export function emailLogin(featheryClient: any) {
   }
 }
 
-export async function sendLoginCode({ fieldVal, servar, methods = null }: any) {
-  methods = methods || servar.metadata.login_methods;
-  if (methods.includes('phone') && phonePattern.test(fieldVal)) {
+export async function sendLoginCode({
+  fieldVal,
+  servar,
+  method
+}: {
+  fieldVal: FeatheryFieldTypes;
+  servar: any;
+  method: 'phone' | 'email';
+}) {
+  const errorPayload = {
+    errorMessage: 'Invalid login',
+    errorField: servar
+  };
+
+  if (typeof fieldVal !== 'string') {
+    return errorPayload;
+  } else if (method === 'phone' && validators.internationalPhone(fieldVal)) {
```

Comment:
should we surface an error message if the user attempts to trigger login and it fails the validator

Reply:
same for the email validator below

Reply:
We were already checking if the email was valid in Form.tsx and displaying an error if it wasn't, so I went ahead and did the same for phone as well. Then I just removed the validation checks from here - we didn't seem to be doing anything in the original code with the error object returned from here anyway

let me know what you think of the change